x-pack/plugins/monitoring/server/lib/elasticsearch/verify_alerting_security.ts

```diff
@@ -43,7 +43,7 @@ export class AlertingSecurity {
 
     return {
       isSufficientlySecure: !isSecurityEnabled || (isSecurityEnabled && isTLSEnabled),
-      hasPermanentEncryptionKey: !encryptedSavedObjects?.usingEphemeralEncryptionKey,
+      hasPermanentEncryptionKey: Boolean(encryptedSavedObjects),
     };
```

Comment:
__optional nit:__ after looking at the code that uses `isSufficientlySecure` and `hasPermanentEncryptionKey` it seems you can just include `Boolean(encryptedSavedObjects)` condition to `isSufficientlySecure` and use only this property everywhere. But up to you, I hope in the future alerting plugin will expose something like this instead so that you can remove explicit dependency on `encryptedSavedObjects`.

Reply:
Gotcha, yea this is pretty much a copy/paste from the alerting code, [from here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/alerts/server/routes/health.ts) so I'd honestly defer to their judgement here too. We just need a way to detect this scenario and let users know